docs/style-guide.adoc

```diff
@@ -181,6 +181,23 @@ WITH rand() AS rand, {param} AS map
 RETURN rand, map.propertyKey, count(n)
 ----
 
+. Write boolean literals in lower case.
++
+.Bad
+[source, cypher]
+----
+WITH TRUE AS b1, False AS b2
+RETURN b1 && b2
+----
+.. No padding space for parameters.
```

Comment:
I think this sentence may be an error? At the moment, we're trying to show that boolean lits must be in lower case...